packages/ui/src/theme/css/component/fileUploader.scss

```diff
@@ -100,11 +100,15 @@
         --amplify-components-fileuploader-file-image-border-radius
       );
       color: var(--amplify-components-fileuploader-file-image-color);
+      img {
+        max-height: 100%;
```

Comment:
is this needed? the base.scss already sets this for all imgs.

Reply:
It's defaulting to `auto` and it needs to be 100% for it to fill the container.

Reply:
Does inherit do the trick? 100% may cause issues if the parent's width is specified

Reply:
In inherit does not work. It overflows on that value. I can change it to `height` 100% if that's better.